core/src/main/java/io/questdb/griffin/SqlCompiler.java

```diff
@@ -92,6 +92,8 @@
     private final TextLoader textLoader;
     private final FilesFacade ff;
 
+    static final double MAX_FLOAT_VALUE = Float.MAX_VALUE;
```

Comment:
These two are declared but not used?

Reply:
Thanks for pointing this out. I'll remove them .